policies/cla.md

```diff
@@ -0,0 +1,65 @@
+# [ITUGLIB](/)
+
+## GMake - GUARDIAN GNUMake Port
+
+## Contributor Agreements
+
+Every non-trivial contribution to ITUGLIB needs to be
+covered by a signed valid enforceable
+Contributor License Agreement (CLA) from all original authors
+especially if you are contributing other people's changes.
+Our policy is based on the general practices of other organizations
+include [the Apache Software Foundation](https://www.apache.org).
+Their CLA policy is available [here](https://www.apache.org/licenses/#clas)
+The ITUGLIB policy is:
+
+ITUGLIB requires that all contributors of significant or non-trivial ideas, code, or
+documentation complete, sign, and submit (via email) an
+[Contributor License Agreement (CLA)](ituglib_cla.pdf).
```

Comment:
Where is this file?

Reply:
We don't have one yet. I sent a sample to @xidbhonaker for adaptation.

Reply:
OK, so once we put it next to the CLA, this link should work